bin/fetch_latest

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
I'd like to have a script to automatically update the file, if a new one is found.
I didn't have time to finish this. shall we merge it anyways (it can tell you if a newer version is out) or not?